### DISCUSSION TITLE
(Appeal) I was banned from discord

### DISCUSSION THREAD

**Discussion by EstebanJaraDev** [pterodactyl/panel#5207] [Support]
Aquí tienes la traducción al inglés:

Hello, I'm from Costa Rica, and I've been banned from Discord twice. The second time was by mistake. Please review this issue. I am trying to create my first hosting company using Pterodactyl. Since I didn't have information and didn't know the rules, I asked about software that wasn't part of the community, and I was expelled. A user told me not to mention software unrelated to Pterodactyl, so I started researching and was recommended WHMCS. For the second time, I made an inquiry about something related to Pterodactyl and WHMCS.

I just logged into Discord to check and continue my setup, only to find out that I was banned again. I don't see warnings being issued, and I believe it would be more professional if a message was deleted and perhaps a private message was sent with the reason, so one could avoid repeating the mistake and know what happened. This would be very helpful since I am trusting you and using your software. I would appreciate your assistance, as I want to pursue this project to move forward in life.

I am looking for people interested in creating a Gaming Hosting project, with knowledge in programming, graphic design, marketing, etc.

User: iencoding
ID: 839736061647126538

**Comment by EstebanJaraDev** [comment]
Hi, can someone from the staff team help me? I'm stuck with installing pterodactyl for my project and I can't continue because I got banned without giving me a warning or something.

**Reply by danny6167** [reply]
Hi. It looks like you've been unbanned already.
Please let me know if that isn't the case.
